clang/test/Driver/hipstdpar.c

```diff
@@ -1,21 +1,17 @@
-// REQUIRES: x86-registered-target
-// REQUIRES: amdgpu-registered-target
-// REQUIRES: system-linux
```

Comment:
I'm not sure if the removal of `system-linux` would cause issue.

Reply:
This is a pile of workarounds, there's no reason any of these tests should be host dependent